release_notes/v2.1.0.txt

```diff
@@ -0,0 +1,17 @@
+v2.1.0
+------
+
+Release Notes
+-------------
+There are minimal changes between v2.1.0 and v2.0.0, please see the change log for a full list of updates.
+
+The release-2.0 branch has been renamed to release-2.x; the v2.1.0 release supercedes v2.0.0.
+A release-2.lts branch will be created when the 2.x LTS release has been decided, please see the proposed Fabric LTS strategy: 
```

Comment:
I'd be tempted to state that this is not an LTS release and be a bit more vague about the future. Perhaps linking to the current LTS release as well as the LTS RFC

Reply:
Think Line 10 is the LTS RFC?